tests/fixtures/nina/sample_warnings.json

```diff
@@ -0,0 +1,44 @@
+[
```

Comment:
Move to tests/components/nina/fixtures/sample_warnings.json

See comment in test.